test-it/karma/publicapi/ReadJson.Test.js

```diff
@@ -0,0 +1,640 @@
+describe('The Vireo EggShell readJSON api can read', function () {
+    'use strict';
+    // Reference aliases
+    var Vireo = window.NationalInstruments.Vireo.Vireo;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+
+    var vireo = new Vireo();
+
+    var publicApiMultipleTypesViaUrl = fixtures.convertToAbsoluteFromFixturesDir('publicapi/MultipleTypes.via');
+
+    beforeAll(function (done) {
+        fixtures.preloadAbsoluteUrls([
+            publicApiMultipleTypesViaUrl
+        ], done);
+    });
+
+    var viName = 'MyVI';
+    beforeAll(function () {
+        vireoRunner.rebootAndLoadVia(vireo, publicApiMultipleTypesViaUrl);
+    });
+
+    describe('scalars of type', function () {
+        it('Boolean', function () {
+            var actualJSON = vireo.eggShell.readJSON(viName, 'dataItem_Boolean');
+            var actual = JSON.parse(actualJSON);
+            expect(actual).toBe(true);
+        });
+
+        it('String', function () {
+            var actualJSON = vireo.eggShell.readJSON(viName, 'dataItem_String');
+            var actual = JSON.parse(actualJSON);
+            expect(actual).toBe('Hello');
+        });
+
+        it('Double', function () {
+            var actualJSON = vireo.eggShell.readJSON(viName, 'dataItem_NumericDouble');
+            var actual = JSON.parse(actualJSON);
+            expect(actual).toBe(123.456);
```

Comment:
This isn't the result of a mathematical operation. We are writing an IEEE754 value to memory and reading it back. We should expect exact representation serializing IEEE754 values in and out.